OpenEdXMobile/src/main/java/org/edx/mobile/view/custom/error/EdxCourseAccessErrorState.kt

```diff
@@ -49,6 +49,17 @@ class EdxCourseAccessErrorState @JvmOverloads constructor(
                 layout.secondaryButton.root.setVisibility(true)
                 layout.secondaryButton.root.text = context.getText(R.string.label_find_a_course)
             }
+            State.NOT_STARTED -> {
+                layout.heading.text = context.getString(R.string.course_not_started)
+                layout.description.text =
+                    context.getString(R.string.message_course_not_started, date)
+                layout.layoutUpgradeFeatures.root.setVisibility(false)
```

Comment:
we can use `replacePrimaryWithSecondaryButton(resId: String)` where we are only using static primary button.
same in `State.AUDIT_ACCESS_EXPIRED`.